vita3k/gui/src/home_screen.cpp

```diff
@@ -257,6 +257,8 @@ static bool app_filter(const std::string &app) {
         if (!filter_app_region({ "PCS" }))
             return true;
         break;
+    case ALL:
+        return true;
```

Comment:
```suggestion
    case ALL:
        break;
```
if you adding it for any compile warning because maybe all enum is not set ? using this, for directly go outside switch and return false in end

Reply:
and set this case in first, for respect order of enum 👌

Reply:
also, if you adding all here, delete: if (app_region != ALL) in 554 line, no needed anymore